UKHomeOffice/sgmr-service
# NMSW-511 Update wording on submission confirmation

### AC / Description of changes

See attachments on Jira ticket for changes to make to receipt.

The tracked document shows the changes

The Cleansed document is the final version

### To Test

- Run app
- Sign in
- Go to /save-voyage/voyage-submitted
- > The content should be updated to match what is in the attachment

### Notes

---

**Remember to merge to feature branches not `master`.**
